更新源码eslint规范

zhanxu33 opened the pull request:
更新源码eslint规范

tencentblueking-adm commented:
[![CLA assistant check](https://bk-cla.bktencent.com:443/pull/badge/not_signed)](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-lesscode?pullRequest=1225) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-lesscode?pullRequest=1225) before we can accept your contribution.<br/>**5** out of **6** committers have signed the CLA.<br/><br/>:white_check_mark: terlinhe<br/>:white_check_mark: ielgnaw<br/>:white_check_mark: forrany<br/>:white_check_mark: zhanxu33<br/>:white_check_mark: luofann<br/>:x: LivySara<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-lesscode?pullRequest=1225) it.</sub>

LivySara committed:
feat: vue2.6.x升级到vue2.7.x

LivySara committed:
feat: vue2组件缺失的属性补充
# Reviewed, transaction id: 17232

forrany committed:
添加 vue3 & 修复交互问题
# Reviewed, transaction id: 17352

LivySara committed:
chore: 引入@vue/composition-api改为vue
# Reviewed, transaction id: 17557

terlinhe committed:
Merge pull request #1348 from LivySara/vue2_update

vue2.6升级&vue2组件属性补充

terlinhe committed:
Merge pull request #1349 from forrany/feat_integration_bkflow

添加 vue3 & 修复交互问题

LivySara committed:
feat: 基于vue2的组件库事件displayName配置添加
# Reviewed, transaction id: 17908

luofann committed:
fix: 流程验收问题修复

luofann committed:
minor: 任务列表支持按创建人和创建时间筛选
# Reviewed, transaction id: 17932

terlinhe committed:
Merge pull request #1350 from luofann/feat_integration_bkflow

fix: 流程验收问题修复

LivySara committed:
feat: 基于vue3的组件库事件displayName配置添加
# Reviewed, transaction id: 17960

terlinhe committed:
Merge pull request #1351 from LivySara/dev_local

pc端vue2/3的组件事件的displayName添加

luofann committed:
minor: 国际化修改
# Reviewed, transaction id: 18544

LivySara committed:
feat: vue2组件属性分类
# Reviewed, transaction id: 18763

terlinhe committed:
Merge pull request #1352 from LivySara/dev_local

feat: vue2组件属性分类

LivySara committed:
fix: 属性名样式
# Reviewed, transaction id: 18931

LivySara committed:
fix: 补充vue2表单容器组件属性分类
# Reviewed, transaction id: 18932

LivySara committed:
feat: vue3组件属性分类
# Reviewed, transaction id: 18933

terlinhe committed:
Merge branch 'develop' of https://github.com/TencentBlueKing/bk-lesscode into feat_config

terlinhe committed:
fix: 修改cheerio版本

terlinhe committed:
feat: 文案规范修改

LivySara committed:
feat: 插槽配置渲染页面修改

LivySara committed:
feat: vue2组件插槽配置belongGroup添加

LivySara committed:
feat: vue3组件插槽配置belongGroup添加
# Reviewed, transaction id: 19066

terlinhe committed:
Merge pull request #1353 from LivySara/dev_local

样式修正以及vue组件属性分类

terlinhe committed:
feat: 使用chrome浏览器提示
# Reviewed, transaction id: 19080

ielgnaw committed:
Merge pull request #1354 from terlinhe/feat_config

feat: 使用chrome浏览器提示

forrany committed:
flowManage 组件适配下载和部署

terlinhe committed:
Merge pull request #1355 from forrany/feat_integration_bkflow

flowManage 组件适配下载和部署

LivySara committed:
fix: 插槽和属性分组UI细节修改
# Reviewed, transaction id: 19113

terlinhe committed:
Merge pull request #1356 from LivySara/dev_local

fix: 插槽和属性分组UI细节修改

LivySara committed:
fix: vue2组件插槽和属性分组修改

LivySara committed:
fix: vue3组件插槽和属性分组修改

LivySara committed:
feat: 表单、表单容器&数据管理容器属性面板样式调整

LivySara committed:
feat: 页面&表单容器内表单项属性配置UI调整
# Reviewed, transaction id: 19230

terlinhe committed:
Merge pull request #1357 from LivySara/dev_local

分组名称&分组样式以及交互修改

LivySara committed:
fix: 页面开发中配置页面样式进行微调
# Reviewed, transaction id: 19335

terlinhe committed:
Merge pull request #1358 from LivySara/dev_local

fix: 页面开发中配置页面样式进行微调

LivySara committed:
fix: 搜索属性时进入页面条件修改
# Reviewed, transaction id: 19497

terlinhe committed:
Merge pull request #1359 from LivySara/dev_local

fix: 搜索属性时进入页面条件修改

LivySara committed:
feat: vue2的vant组件属性和插槽分组

LivySara committed:
feat: vue3的vant组件属性和插槽分组

luofann committed:
Merge branch 'lesscode-nocode' into feat_integration_bkflow

LivySara committed:
feat: echarts配置部分简化
# Reviewed, transaction id: 19799

terlinhe committed:
feat: 处理bktooltips默认allowHTML属性
# Reviewed, transaction id: 19826

terlinhe committed:
feat: 兼容当组件的数据中，含有连字符跟驼峰格式的的相同属性 --story=119941102
# Reviewed, transaction id: 19860

terlinhe committed:
Merge branch 'develop' of https://github.com/TencentBlueKing/bk-lesscode into feat_config

terlinhe committed:
Merge pull request #1360 from LivySara/dev_local

feat: vant组件属性和插槽分组和echarts配置部分简化

terlinhe committed:
fix: project id置空
# Reviewed, transaction id: 19918

ielgnaw committed:
Merge pull request #1361 from terlinhe/feat_config

fix: 问题修复

terlinhe committed:
fix: 新增数据入库时置空id字段

terlinhe committed:
fix: 不返回codeurl

terlinhe committed:
fix:  部署后iamappid
# Reviewed, transaction id: 20389

ielgnaw committed:
Merge pull request #1363 from terlinhe/feat_config

fix: 问题修复

zhanxu33 committed:
优化AI使用模板的流程
# Reviewed, transaction id: 20429

terlinhe committed:
Merge pull request #1364 from zhanxu33/develop

优化AI使用模板的流程

luofann committed:
refactor: 数据管理容器详情
# Reviewed, transaction id: 20488

luofann committed:
fix: 流程跳转问题修复
# Reviewed, transaction id: 20496

terlinhe committed:
Merge pull request #1365 from luofann/feat_integration_bkflow

新版流程问题处理

luofann committed:
refactor: 移动端页面去掉表单编辑器相关菜单模块
# Reviewed, transaction id: 20502

terlinhe committed:
Merge pull request #1366 from luofann/feat_integration_bkflow

refactor: 移动端页面去掉表单编辑器相关菜单模块

luofann committed:
refactor: 画布元素事件绑定执行流程操作增加部署状态逻辑相关优化
# Reviewed, transaction id: 20517

LivySara committed:
feat: 模板中应用变量和函数资源脱离项目

LivySara committed:
feat: 模板市场未添加应用不可拖拽
# Reviewed, transaction id: 20519

terlinhe committed:
Merge pull request #1367 from luofann/feat_integration_bkflow

refactor: 画布元素事件绑定执行流程操作增加部署状态逻辑相关优化

LivySara committed:
feat: 市场模板直接预览时需传递变量和函数
# Reviewed, transaction id: 20540

LivySara committed:
feat: 模板市场直接下载源码时需传递变量和函数
# Reviewed, transaction id: 20547

terlinhe committed:
Merge pull request #1362 from LivySara/dev_local

feat: 模板中应用函数和变量资源脱离项目

terlinhe committed:
fix: 新增时id置空问题修复
# Reviewed, transaction id: 20582

ielgnaw committed:
Merge pull request #1368 from terlinhe/feat_config

fix: 新增时id置空问题修复

LivySara committed:
fix: newFuncGroups传值修改
# Reviewed, transaction id: 20655

terlinhe committed:
Merge pull request #1369 from LivySara/dev_local

fix: newFuncGroups传值修改

luofann committed:
feat: 数据处理节点支持选择流程前置节点中的表单变量
# Reviewed, transaction id: 20691

forrany committed:
update: 流程添加分页 & formId 逻辑修改

forrany committed:
fix: 修复节点表单出现 undefined 接口的问题

luofann committed:
refactor: 流程变量值替换
# Reviewed, transaction id: 20704

terlinhe committed:
Merge pull request #1371 from forrany/feat_integration_bkflow

Feat integration bkflow

terlinhe committed:
Merge pull request #1370 from luofann/feat_integration_bkflow

feat: 数据处理节点支持选择流程前置节点中的表单变量

luofann committed:
refator: 增加系统自动添加数据处理节点逻辑
# Reviewed, transaction id: 20834

terlinhe committed:
Merge pull request #1372 from luofann/feat_integration_bkflow

refator: 增加系统自动添加数据处理节点逻辑

luofann committed:
feat: 流程详情页增加url中的流程id是否属于当前项目维度的判断
# Reviewed, transaction id: 20850

luofann committed:
fix: 请求参数错误修复
# Reviewed, transaction id: 20851

luofann committed:
minor: 流程通知方式去掉企微机器人
# Reviewed, transaction id: 20853

terlinhe committed:
Merge pull request #1373 from luofann/feat_integration_bkflow

feat: 流程详情页增加url中的流程id是否属于当前项目维度的判断

luofann committed:
refactor: 网关回写接口去掉项目鉴权装饰器
# Reviewed, transaction id: 20926

terlinhe committed:
Merge pull request #1374 from luofann/feat_integration_bkflow

refactor: 网关回写接口去掉项目鉴权装饰器

luofann committed:
refator: 旧流程自动创建表单时表名统一做小写处理
# Reviewed, transaction id: 20993

ielgnaw committed:
Merge pull request #1375 from luofann/feat_integration_bkflow

refator: 旧流程自动创建表单时表名统一做小写处理

luofann committed:
feat: 流程编辑页测试执行功能
# Reviewed, transaction id: 21004

luofann committed:
minor: 联调环境网关stage取值修改
# Reviewed, transaction id: 21005

terlinhe committed:
Merge pull request #1376 from luofann/feat_integration_bkflow

feat: 流程编辑页测试执行功能

luofann committed:
feat: 测试执行流程节点处理面板
# Reviewed, transaction id: 21070

luofann committed:
minor: 去掉debug代码
# Reviewed, transaction id: 21073

terlinhe committed:
Merge pull request #1377 from luofann/feat_integration_bkflow

feat: 测试执行流程节点处理面板

luofann committed:
fix: 退出测试执行流程
# Reviewed, transaction id: 21081

terlinhe committed:
Merge pull request #1378 from luofann/feat_integration_bkflow

fix: 退出测试执行流程

LivySara committed:
fix: 画布、saas页面卸载时释放锁调整
# Reviewed, transaction id: 21127

terlinhe committed:
Merge pull request #1379 from LivySara/dev_local

fix: 画布、saas页面卸载时释放锁调整

LivySara committed:
fix: 同一属性的短线和驼峰命名的值转换
# Reviewed, transaction id: 21226

terlinhe committed:
Merge pull request #1380 from LivySara/dev_local

fix: 同一属性的短线和驼峰命名的值转换

luofann committed:
fix: 流程测试相关问题修复
# Reviewed, transaction id: 21311

terlinhe committed:
Merge pull request #1381 from luofann/feat_integration_bkflow

fix: 流程测试相关问题修复

luofann committed:
feat: 应用部署对接流程
# Reviewed, transaction id: 21488

luofann committed:
merge branch
# Reviewed, transaction id: 21493

luofann committed:
feat: 应用部署调试
# Reviewed, transaction id: 21505

luofann committed:
feat: 流程应用部署调试
# Reviewed, transaction id: 21536

luofann committed:
feat: 应用部署对接流程调试
# Reviewed, transaction id: 21542

terlinhe committed:
Merge pull request #1382 from luofann/feat_integration_bkflow

feat: 应用部署对接流程

luofann committed:
refactor: 平台部署的应用在引擎服务收敛为一个空间
# Reviewed, transaction id: 21549

luofann committed:
refactor: 源码eslint规范修复
# Reviewed, transaction id: 21550

luofann committed:
refactor: 认证信息模板字符串替换
# Reviewed, transaction id: 21551

luofann committed:
refactor: 应用源码分别适配vue2/3
# Reviewed, transaction id: 21552

terlinhe committed:
Merge pull request #1383 from luofann/feat_integration_bkflow

refactor: 平台部署的应用在引擎服务收敛为一个空间

luofann committed:
refactor: 应用生成源码修改
# Reviewed, transaction id: 21867

luofann committed:
refactor: vue3源码render修改
# Reviewed, transaction id: 21894

ielgnaw committed:
Merge pull request #1384 from luofann/feat_integration_bkflow

refactor: 应用生成源码修改

luofann committed:
fix: vue3应用流程数据管理容器render函数语法配置修复
# Reviewed, transaction id: 21904

luofann committed:
refactor: 去掉多余的国际化
# Reviewed, transaction id: 21906

ielgnaw committed:
Merge pull request #1385 from luofann/feat_integration_bkflow

fix: vue3应用流程数据管理容器render函数语法配置修复

luofann committed:
refactor: 应用生成网关增加新版流程判断
# Reviewed, transaction id: 21993

luofann committed:
refactor: 去掉多余空格
# Reviewed, transaction id: 21995

terlinhe committed:
Merge pull request #1386 from luofann/feat_integration_bkflow

refactor: 应用生成网关增加新版流程判断

luofann committed:
refactor: 部署应用时网关为自己授权
# Reviewed, transaction id: 22048

terlinhe committed:
Merge pull request #1387 from luofann/feat_integration_bkflow

refactor: 部署应用时网关为自己授权

forrany committed:
fix: 体验问题修复（文案优化）

fix: delete console

terlinhe committed:
Merge pull request #1388 from forrany/feat_integration_bkflow

fix: 体验问题修复（文案优化）

luofann committed:
fix: 修复部署应用的流程数据管理容器问题
# Reviewed, transaction id: 22175

luofann committed:
refactor: 去掉多余逗号
# Reviewed, transaction id: 22176

terlinhe committed:
Merge pull request #1389 from luofann/feat_integration_bkflow

fix: 修复部署应用的流程数据管理容器问题

terlinhe committed:
feat: 体验问题修复
# Reviewed, transaction id: 22515

forrany committed:
feat: 允许用户自定义表名并在表单选择对话框中显示表名和表名提示信息

terlinhe committed:
Merge pull request #1390 from forrany/feat_integration_bkflow

feat: 允许用户自定义表名并在表单选择对话框中显示表名和表名提示信息

luofann committed:
feat: 流程表单画布保存是支持自定义输入数据表表名
# Reviewed, transaction id: 22805

LivySara committed:
fix: 编辑页面属性面板样式修复
# Reviewed, transaction id: 22917

luofann committed:
feat: 人工节点数据表表名默认值修改
# Reviewed, transaction id: 22851

terlinhe committed:
feat: 体验问题修复
# Reviewed, transaction id: 22921

luofann committed:
refactor: 数据表表名相关问题优化
# Reviewed, transaction id: 22957

terlinhe committed:
Merge pull request #1392 from LivySara/dev_local

fix: 编辑页面属性面板样式修复

luofann committed:
refactor: 数据管理容器数据表可选项显示名称优化
# Reviewed, transaction id: 23007

LivySara committed:
fix: 应用模板时添加framework字段
# Reviewed, transaction id: 23018

terlinhe committed:
Merge pull request #1395 from LivySara/dev_local

fix: 应用模板时添加framework字段

luofann committed:
refactor: 部署应用在bkflow按单独空间隔离
# Reviewed, transaction id: 23056

luofann committed:
feat: lesscode网关apiname支持通过环境变量配置
# Reviewed, transaction id: 23065

luofann committed:
refactor: 取应用配置文件修改
# Reviewed, transaction id: 23109

luofann committed:
解决代码冲突
# Reviewed, transaction id: 23159

terlinhe committed:
feat: v1.1.9 changelog
# Reviewed, transaction id: 23170

terlinhe committed:
Merge pull request #1398 from luofann/feat_integration_bkflow

合并新版流程对接代码到develop分支

ielgnaw committed:
Merge pull request #1399 from terlinhe/feat_config

feat: v1.1.9 changelog

luofann committed:
refactor: 去掉vue composition-api模块的导入
# Reviewed, transaction id: 23187

terlinhe committed:
Merge pull request #1400 from luofann/feat_integration_bkflow

refactor: 去掉vue composition-api模块的导入

luofann committed:
fix: 应用源码网关访问环境改为从环境变量读取
# Reviewed, transaction id: 23213

luofann committed:
refactor: 应用源码babel依赖声明版本支持npm8以下版本

terlinhe committed:
Merge pull request #1402 from luofann/feat_integration_bkflow

应用源码依赖配置相关修改适配

luofann committed:
feat: 创建bkflow空间时传入管理员信息
# Reviewed, transaction id: 23313

terlinhe committed:
fix: 兼容旧数据中renderEvents缺少新属性的问题
# Reviewed, transaction id: 23321

ielgnaw committed:
Merge pull request #1405 from terlinhe/feat_config

fix: 兼容旧数据中renderEvents缺少新属性的问题

forrany committed:
fix: 修复移动端模板预览的问题

terlinhe committed:
Merge pull request #1406 from forrany/bugfix/mobile-preview1

fix: 修复移动端模板预览的问题

LivySara committed:
fix: vue3表格物料部分属性未生效修复
# Reviewed, transaction id: 23425

terlinhe committed:
Merge pull request #1407 from LivySara/dev_local

fix: vue3表格物料部分属性未生效修复

terlinhe committed:
fix: 设置应用公开时异常问题

terlinhe committed:
feat: 数据表api增加函数示例
# Reviewed, transaction id: 23527

forrany committed:
feat(移动布局): 增强移动页面容器功能 & 优化交互

- 优化新增页面的插入逻辑
- 调整页面组件的样式以适应新增和删除功能
- 为每个swiper slide添加Vue指令支持以实现动态绑定
# Reviewed, transaction id: 24868

forrany committed:
feat(flow-manage-container): 添加图标 & 优化交互和UI

terlinhe committed:
Merge pull request #1408 from forrany/bugfix/flow-style

feat(flow-manage-container): 添加图标 & 优化交互和UI

terlinhe committed:
Merge pull request #1403 from luofann/feat_integration_bkflow

feat: 创建bkflow空间时传入管理员信息

terlinhe committed:
feat: 表单容器文案修改
# Reviewed, transaction id: 23817

terlinhe committed:
merge
# Reviewed, transaction id: 23818

ielgnaw committed:
Merge pull request #1410 from terlinhe/feat_config

feat: 体验问题修复 & 数据源api增加函数示例

terlinhe committed:
feat: 页面配置增加pagecode字段展示
# Reviewed, transaction id: 23852

forrany committed:
refactor(project-code):  增加对数据库中未设置框架的兼容性处理 & 默认为 vue2

terlinhe committed:
Merge pull request #1411 from forrany/bugfix/flow-style

refactor(project-code):  增加对数据库中未设置框架的兼容性处理 & 默认为 vue2

forrany committed:
refactor(preview): 增加移动端预览的高度调整逻辑【仅在移动端预览时，设置 swiper-container 的高度】
# Reviewed, transaction id: 23925

LivySara committed:
fix: trycatch时响应内容用ctx.send
# Reviewed, transaction id: 24100

terlinhe committed:
Merge branch 'develop' of https://github.com/TencentBlueKing/bk-lesscode into feat_config

LivySara committed:
fix: vue3中table列配置隐藏多选框&索引
# Reviewed, transaction id: 24788

terlinhe committed:
Merge pull request #1414 from LivySara/dev_local

table列配置&函数捕获错误修复

terlinhe committed:
fix: sideslider默认层级问题
# Reviewed, transaction id: 24805

ielgnaw committed:
Merge pull request #1415 from terlinhe/feat_config

fix: sideslider默认层级问题

terlinhe committed:
feat: 更新chrometips版本
# Reviewed, transaction id: 24832

terlinhe committed:
Merge branch 'develop' of https://github.com/TencentBlueKing/bk-lesscode into feat_config

terlinhe committed:
Merge pull request #1412 from forrany/bugfix/flow-style

refactor(preview): 增加移动端预览的高度调整逻辑【仅在移动端预览时，设置 swiper-container 的高度】

terlinhe committed:
Merge pull request #1413 from forrany/bugfix/mobile-preview1

 feat(移动布局): 增强移动页面容器功能 & 优化交互 & 支持签到模板的交互

forrany committed:
fix(preview): 修复移动端预览Swiper高度设置
# Reviewed, transaction id: 24930

terlinhe committed:
Merge pull request #1416 from forrany/bugfix/mobile-preview1

fix(preview): 修复移动端预览Swiper高度设置

LivySara committed:
feat: 帮助文档国际化

terlinhe committed:
fix：体验问题修复

LivySara committed:
fix: vue3物料组件form的rules属性误删复原
# Reviewed, transaction id: 25193

terlinhe committed:
feat: 内置h5 api

zhanxu33 committed:
优化AI小鲸组件
# Reviewed, transaction id: 25529

terlinhe committed:
Merge pull request #1418 from zhanxu33/develop

优化AI小鲸组件

zhanxu33 committed:
AI小鲸组件优化

terlinhe committed:
Merge pull request #1419 from zhanxu33/develop

AI小鲸组件优化

zhanxu33 committed:
ai小鲸组件优化

terlinhe committed:
Merge pull request #1420 from zhanxu33/develop

ai小鲸组件优化

LivySara committed:
fix: 组件属性json编辑弹框禁止触发快捷操作

LivySara committed:
feat: 模板市场按框架分类过滤

LivySara committed:
fix: vue3的table组件属性border配置修改
# Reviewed, transaction id: 25971

zhanxu33 committed:
修改导出的date格式
# Reviewed, transaction id: 26043

terlinhe committed:
Merge pull request #1422 from zhanxu33/develop

修改导出的date格式

LivySara committed:
fix: 帮助文档专有名词英文翻译修改
# Reviewed, transaction id: 26053

LivySara committed:
fix: 应用统一翻译成APP

terlinhe committed:
Merge pull request #1417 from LivySara/dev_local

帮助文档国际化以及vue3物料form得rules属性误删复原

LivySara committed:
fix: 帮助文档详情样式放开

LivySara committed:
fix: vue3/vue2的bk组件带有vModel的属性tips修改&vue3的bk组件可配置vModel指令添加
# Reviewed, transaction id: 26408

forrany committed:
fix(sideslider): 修复 transfer 属性静态值问题

terlinhe committed:
Merge pull request #1424 from forrany/bugfix/transter-props

fix(sideslider): 修复 transfer 属性静态值问题

zhanxu33 committed:
关闭小鲸划词功能
# Reviewed, transaction id: 26488

terlinhe committed:
Merge pull request #1425 from zhanxu33/develop

关闭小鲸划词功能

LivySara committed:
fix: 源码生成公共样式优化
# Reviewed, transaction id: 26745

zhanxu33 committed:
升级 @blueking/ai-blueking

terlinhe committed:
Merge pull request #1426 from zhanxu33/develop

升级 @blueking/ai-blueking

terlinhe committed:
Merge branch 'develop' of https://github.com/TencentBlueKing/bk-lesscode into feat_config

terlinhe committed:
Merge branch 'develop' of https://github.com/TencentBlueKing/bk-lesscode into feat_config
# Reviewed, transaction id: 27311

LivySara committed:
fix: vue3侧边导航栏二级路由触发click事件停止冒泡
# Reviewed, transaction id: 27745

terlinhe committed:
feat: 升级部分依赖跟预览vue3模板引入http
# Reviewed, transaction id: 27755

terlinhe committed:
Merge pull request #1423 from LivySara/dev_local

帮助文档样式和vModel绑定修复

LivySara committed:
fix: vModel指令绑定的变量清除时renderValue值清空

LivySara committed:
fix: 在画布上单选框组&多选框组不同方式改变选中值时渲染异常修复

LivySara committed:
fix: vue3的bk组件属性书写错误修复
# Reviewed, transaction id: 28010

terlinhe committed:
Merge pull request #1427 from LivySara/dev_local

组件vModel绑定值修复

LivySara committed:
fix: 兼容vue2在画布上单选框组&多选框组不同方式改变选中值时渲染异常修复
# Reviewed, transaction id: 28021

terlinhe committed:
feat: v1.1.10changelog

terlinhe committed:
fix: h5容器属性重复问题
# Reviewed, transaction id: 28037

terlinhe committed:
feat: 内置h5api
# Reviewed, transaction id: 28039

ielgnaw committed:
Merge pull request #1429 from terlinhe/feat_config

feat: v1.1.10changelog && 问题修复

terlinhe committed:
Merge pull request #1428 from LivySara/dev_local

fix: 兼容vue2在画布上单选框组&多选框组不同方式改变选中值时渲染异常修复

LivySara committed:
fix: 模板市场添加至应用时补充framework参数
# Reviewed, transaction id: 28065

terlinhe committed:
Merge pull request #1430 from LivySara/dev_local

fix: 模板市场添加至应用时补充framework参数

terlinhe committed:
fix: 兼容生成sql差异
# Reviewed, transaction id: 28234

ielgnaw committed:
Merge pull request #1431 from terlinhe/feat_config

fix: 兼容生成sql差异

zhanxu33 committed:
AI体验问题修复
# Reviewed, transaction id: 28242

terlinhe committed:
Merge pull request #1432 from zhanxu33/develop

AI体验问题修复

terlinhe committed:
fix: 生成sql优化
# Reviewed, transaction id: 30623

zhanxu33 committed:
修改AI返回的数据结构
# Reviewed, transaction id: 31767

terlinhe committed:
Merge pull request #1436 from zhanxu33/develop

修改AI返回的数据结构

zhanxu33 committed:
ai 切换成 hunyuan & 修改sql参数
# Reviewed, transaction id: 34704

terlinhe committed:
Merge pull request #1438 from zhanxu33/develop

AI 切换成 hunyuan & 修改sql参数

zhanxu33 committed:
修改sql参数的写法
# Reviewed, transaction id: 34770

terlinhe committed:
Merge pull request #1439 from zhanxu33/develop

修改sql参数的写法

terlinhe committed:
feat: 生成sql优化

terlinhe committed:
fix: vue2 table 多选列时header展示异常
# Reviewed, transaction id: 34780

ielgnaw committed:
Merge pull request #1440 from terlinhe/feat_config

feat: 生成sql优化 &&  fix table 多选列时header展示异常

luofann committed:
refactor: 修改表单容器操作按钮文案

luofann committed:
--story=120157583 流程关联表单页、数据管理页、流程管理页
# Reviewed, transaction id: 39942

luofann committed:
新建绑定更新&源码支持提单触发流程

luofann committed:
feat: 手动创建的数据管理容器关联流程节点
# Reviewed, transaction id: 39946

luofann committed:
refactor: 组件源码国际化
# Reviewed, transaction id: 39947

terlinhe committed:
Merge pull request #1441 from luofann/feat_form_container

--story=120157583 流程关联表单页、数据管理页、流程管理页

zhanxu33 committed:
升级项目babel依赖
# Reviewed, transaction id: 41338
